README.md

```diff
@@ -1 +1,35 @@
-# prerequisite-class
\ No newline at end of file
+# 📦 Prerequisite Class
+
+Before running any of the application templates in this repository, make sure your development environment is properly set up.
+
+---
+
+## ✅ Requirements
+
+To run the applications smoothly, ensure the following are installed on your system:
+
+- 🐳 Docker
+  Used to build and run containerized versions of each app template.  
+
+- 💻 Basic command-line knowledge  
+  You’ll need to be familiar with basic terminal commands for navigating directories and running scripts.
+
+---
+
+## 📥 Installation Guide for Docker
```

Comment:
- เปลี่ยนจาก Installation Guide for Docker
- เป็น Installation Guide for Container